src/kili/services/export/format/coco/__init__.py

```diff
@@ -157,113 +168,226 @@ def _convert_kili_semantic_to_coco(
     )
 
     # Mapping category - category id
-    cat_kili_id_to_coco_id = _get_kili_cat_id_to_coco_cat_id_mapping(job)
-    labels_json["categories"] = _get_coco_categories(cat_kili_id_to_coco_id)
-    labels_json["images"], labels_json["annotations"] = _get_coco_images_and_annotations(
-        job_name, assets, cat_kili_id_to_coco_id, project_input_type, annotation_modifier
+    cat_kili_id_to_coco_id, labels_json["categories"] = _get_coco_categories_with_mapping(
+        jobs, merged
     )
 
-    label_file_name = output_dir / job_name / "labels.json"
-    label_file_name.parent.mkdir(parents=True, exist_ok=True)
-    with (output_dir / job_name / "labels.json").open("w") as outfile:
-        json.dump(labels_json, outfile)
+    if merged:
+        labels_json["images"], labels_json["annotations"] = _get_coco_images_and_annotations(
+            jobs,
+            assets,
+            cat_kili_id_to_coco_id,
+            project_input_type,
+            annotation_modifier,
+            is_single_job=False,
+        )
+
+        label_file_name = output_dir / "labels.json"
+        label_file_name.parent.mkdir(parents=True, exist_ok=True)
+        with label_file_name.open("w") as outfile:
+            json.dump(labels_json, outfile)
+        label_filenames = [label_file_name]
+    else:  # split case
+        label_filenames = []
+        for job_name, job in jobs.items():
+            labels_json["images"], labels_json["annotations"] = _get_coco_images_and_annotations(
+                {job_name: job},
+                assets,
+                cat_kili_id_to_coco_id,
+                project_input_type,
+                annotation_modifier,
+                is_single_job=True,
+            )
```

Comment:
I'm not sure if it's necessary to add this argument "is_single_job". We can just do `is_single_job = len(jobs) == 1` in the methods called later on maybe

Reply:
the "merged" syntax outputs categories name  "job/category_name", while the "split" syntax outputs "category_name", so I think we should still tell the function which layout it is.